index.js

```diff
@@ -16,15 +21,56 @@ async function run() {
     const logConfigurationOptions = core.getInput("log-configuration-options", { required: false });
     const dockerLabels = core.getInput('docker-labels', { required: false });
     const command = core.getInput('command', { required: false });
+    
+     //New inputs to fecth task definition 
+     const taskDefinitionArn = core.getInput('task-definition-arn', { required: false });
+     const taskDefinitionFamily = core.getInput('task-definition-family', { required: false });
+     const taskDefinitionRevision = Number(core.getInput('task-definition-revision', { required: false}));
 
-    // Parse the task definition
-    const taskDefPath = path.isAbsolute(taskDefinitionFile) ?
-      taskDefinitionFile :
-      path.join(process.env.GITHUB_WORKSPACE, taskDefinitionFile);
-    if (!fs.existsSync(taskDefPath)) {
-      throw new Error(`Task definition file does not exist: ${taskDefinitionFile}`);
+
+    if(taskDefinitionFile && taskDefinitionArn && taskDefinitionFamily){ 
+      core.warning("Task definition file will be option used.");
+    }
+    if(taskDefinitionFamily && !taskDefinitionRevision){
+      core.warning("The latest revision of the task definition family will be provided");
     }
-    const taskDefContents = require(taskDefPath);
+    if(taskDefinitionRevision){
```

Comment:
Should this condition check that **only** taskDefinitionRevision is provided and  taskDefinitionFamily or taskDefinitionArn or taskDefinitionFile are **not provided**?